sites/pycharm-guide/contents/tips/database-visualization/index.md

```diff
@@ -0,0 +1,20 @@
+---
+date: 2020-09-24
+title: Database Visualization 
+technologies: [django]
+topics: [database]
+author: mm
+subtitle: visualize the structure of a database with diagrams
```

Comment:
Make this Visualize; consider a subtitle that adds more than the title had, perhaps focusing on the benefit.